glitter/blocks/image/templates/blockadmin/glitter_image/imageblock/change_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "blockadmin/change_form.html" %}
+{% load static from staticfiles %}
+{% load glitter_admin_js %}
+
+{% block extrastyle %}
+  <script>
+    var DROP_IMAGE_URL="{% url 'block_admin:glitter_image_imageblock_drop_image' %}";
```

Comment:
Let's use an HTML element with an attribute instead of a global javascript variable.